broker/fragment/store_azure.go

```diff
@@ -167,6 +206,22 @@ func (a *azureBackend) Remove(ctx context.Context, fragment pb.Fragment) error {
 	return err
 }
 
+func GetAzureStorageCredential(coreCredential azcore.TokenCredential) (azblob.TokenCredential, error) {
```

Comment:
My comment you resolved was in reference to this entire routine, not the URL comment.

It's not used anywhere that I can tell. If you do intend for it to be exported, it needs a documenting comment for what it is, what it does, why it exist.

Reply:
Yup, this one I missed as its functionality got replaced by `azidentity.NewClientSecretCredential`. I didn't get a warning about an unused function since it had a capitalized name, which TIL means an exported function. Removed!